source/_docs/metrics.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Metrics in the Site Dashboard
+description: Measure your site's traffic with the Metrics tool, found in the Live environment of the Site Dashboard.
+tags: [performance, logs]
+---
+## Access Metrics
+1. Navigate to **<span class="glyphicons glyphicons-charts"></span> Metrics** within the **<span class="glyphicons glyphicons-cardio"></span> Live** environment of your Site Dashboard.
+2. Toggle available date ranges by clicking **Day**, **Week**, or **Month**:
+  ![Charts for pages served and visits within the Metrics tool of the Site Dashboard](/source/docs/assets/images/dashboard/metrics-graphs.png)
+
+## Available Metrics
+{% include("content/traffic-dl.html")%}
+
+For example, imagine two visitors go to `example.com`. The first visitor exits from the home page, but the second visitor continues to browse two additional pages. This would be tracked as 2 visits and 4 pages served.  
+![Diagram demonstrating how pages served and visits are tracked](/source/docs/assets/images/dashboard/metrics-diagram.png)
```

Comment:
Love the visual. What do you think about having a different color line for one visitor vs the other?